test/test-parse-function.cpp

```diff
@@ -853,7 +882,7 @@ TEST(test_parse, unnamed_function_expression_without_body) {
 
 TEST(test_parse, arrow_function_without_parameter_list) {
   {
-    padded_string code(u8"=> x + y"_sv);
+    padded_string code(u8"(=> x + y)"_sv);
```

Comment:
Must fix: This test is trying to test the statement-starts-with-`=>` code path. Adding parentheses here reduces test coverage. Undo this change.